### ISSUE TITLE
>= 1.0 loss

### ISSUE THREAD

**Issue by badcode6** [rinongal/textual_inversion#98]
https://github.com/nicolai256/Stable-textual-inversion_win/issues/37 . on cpu, the training loss gets stuck from 0.980-1.01. I assume that this issue should be brought here too since the differences between these two repositories are mostly if not completely OS-specific changes

**Comment by badcode6** [comment]
![image](https://user-images.githubusercontent.com/70351049/194326008-6e82f2d5-3afc-4298-86b8-50642dab75cb.png)
It does seem like the learning rate was being set to 0. I passed "--scale_lr false" which seems to have fixed that but loss appears to still be at 1 or 1.01.

example: `Epoch 0:   6%|██▌                                      | 19/303 [05:27<1:21:41, 17.26s/it, loss=1, v_num=0, train/loss_simple_step=1.010, train/loss_vlb_step=0.00473, train/loss_step=1.010, global_step=18.00`

**Comment by badcode6** [comment]
Nevermind for now. I restarted fresh with a new command, not using the old embeddings, and now we're good to go. I think unlocking the learning rate and restarting training helped. Thank you.
